### ISSUE TITLE
[MSFT-39] [Login] Username and Password fields element has no labels

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2761]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Open the source and check the Username and Password fields

```
<input ng-model="vm.username" name="username" placeholder="Username or Email" type="text" required="" class="ng-dirty ng-valid-parse ng-touched ng-empty ng-invalid ng-invalid-required">
```

```
<input id="current-password-input" ng-model="vm.currentPassword" ng-model-options="{allowInvalid: true}" ng-focus="vm.onCPFocus($event)" ng-blur="vm.onCPBlur($event)" name="currentPassword" type="password" placeholder="Password" required="" class="input input-btn ng-valid-parse ng-touched ng-empty ng-invalid ng-invalid-required ng-pristine">
```

**Expected Result**
Ensures every form element has a label

**Actual Result**
Username and Password fields element has no labels

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 3.3.2 Labels or Instructions
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F22

**Comment by mishacucicea** [comment]
The placeholders are in place of labels. Introducing labels, might change the design of the page.

**Comment by crazyk07** [comment]
Duplicate of other changes being made
